pkg/skaffold/survey/survey.go

```diff
@@ -17,28 +17,38 @@ limitations under the License.
 package survey
 
 import (
+	"fmt"
 	"io"
 	"os"
-
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/color"
 )
 
 const (
 	Prompt = `Help improve Skaffold! Take a 10 seconds anonymous survey by running
    $skaffold survey`
```

Comment:
`$skaffold` would normally suggest that it's an environment variable.  I'd drop the `$`.